indico/modules/categories/controllers/management.py

```diff
@@ -512,3 +514,40 @@ def _process(self):
                 self.role.members.add(user)
                 logger.info('User %r added to role %r by %r', user, self.role, session.user)
         return jsonify_data(html=_render_role(self.role, collapsed=False))
+
+
+class RHAddCategoryRoleMembersImportCSV(RHManageCategoryRole):
```

Comment:
lets create a mixin from this so this can be simplified to

    class RHAddCategoryRoleMembersImportCSV(ImportRoleMembersMixin, RHManageCategoryRole):
        pass